draft-demarco-oauth-status-assertions.md

```diff
@@ -247,7 +248,7 @@ the expiration datetime of the Digital Credential;
 - MUST enable the offline use cases by employing validation using
 a cryptographic signature and the cryptographic public key of the
 Credential Issuer.
-- MUST NOT contain personal information about the User who owns
+- SHOULD NOT contain personal information about the User who owns
```

Comment:
```suggestion
- MUST NOT contain personal information about the User who owns
```

Reply:
I suggest something along the lines of "MUST NOT contain information about the data subject (user) that isn't already made available to the validator"

Reply:
hmm, thats undermining one of my use cases.

Lets imagine a truck that can be loaded with different products, and each product has a different weight.

The truck is empty waiting to be loaded, it is loaded with copper ore, it drives over a weighing station which creates a status assertion for the truck that asserts: copper, weight.

The rest of the truck credential may have an "unloaded weight", but it won't have "copper".

Let me give a personal credentials use case that centers around new PII disclosures as well.

Lets say a person has no children when their credential is issued, but a few days later then have twins.

The status assertion could be: children: true, or children: 2 ... in either case, it would be new information to a verifier.

If your point is that "determining what is PII is a can of worms"... I agree.

I might recommend we simple avoid the discussion of PII outside of security considerations, and treat the status assertion as conveying information which the issuer can only provide about the subject after the credential has been issued and bound to the subjects public key... this information could be related to time or location or changes in physical traits, the subject of a credential is not necessarily a natural or legal person... see security considerations section __ for guidance on informed consent for personal information related to natural persons, etc....